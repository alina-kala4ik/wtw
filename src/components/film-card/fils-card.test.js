import React from "react";
import render from "react-test-renderer";
import FilmCard from "./film-card";

const film = {
  "id": 4,
  "name": `keks`,
  "posterImage": `img/the-grand-budapest-hotel-poster.jpg`,
  "previewImage": `img/the-grand-budapest-hotel.jpg`,
  "backgroundImage": `img/the-grand-budapest-hotel-bg.jpg`,
  "backgroundColor": `#ffffff`,
  "videoLink": `https://some-link`,
  "previewVideoLink": `https://some-link`,
  "description": `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustave's friend and protege.`,
  "rating": 8.9,
  "scoresCount": 240,
  "director": `Wes Andreson`,
  "starring": [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`, `Saoirse Ronan`],
  "runTime": 99,
  "genre": `Comedy`,
  "released": 2014,
  "isFavorite": false
};

it(`FilmCard render`, () => {
  const three = render
    .create(
        <FilmCard
          film={film}
          handlerFilmCardHover={() => {}}
        />
    )
    .toJSON();

  expect(three).toMatchSnapshot();
});
