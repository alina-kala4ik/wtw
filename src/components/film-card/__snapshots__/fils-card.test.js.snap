// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmCard render 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      autoPlay={false}
      muted={true}
      poster="img/the-grand-budapest-hotel.jpg"
      src="https://some-link"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      onClick={[Function]}
    >
      keks
    </a>
  </h3>
</article>
`;
